client/hotrod-client/src/main/java/org/infinispan/client/hotrod/impl/protocol/InternalFlag.java

```diff
@@ -0,0 +1,26 @@
+package org.infinispan.client.hotrod.impl.protocol;
+
+/**
+ * Flags which are internal to the protocol and shouldn't be exposed to users.
```

Comment:
Just see my solution above, you should be changing `org.infinispan.client.hotrod.Flag` and add single flag there to define the granularity of expiration parameters. Users should be able to define the granularity of the time.